src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/GeoAnalyzer.java

```diff
@@ -0,0 +1,39 @@
+package edu.pugetsound.mathcs.nlp.architecture_nlp.features;
+
+import edu.pugetsound.mathcs.nlp.kb.KBController;
+import edu.pugetsound.mathcs.nlp.lang.Conversation;
+import edu.pugetsound.mathcs.nlp.lang.Utterance;
+/*
+ * Wrapper for the TextAnalyzer Class. Use for any SPF 
+ * related analysis.
+ * 
+ * @Author Brian Dague
+ * @Version 1.0
+ */
+
+public class GeoAnalyzer {
+	
+	private TextAnalyzer analyzer;
+	private KBController knowledge;
+	
+	/*
+	 * Constructor for the GeoAnalyzer class
+	 * @param directory Directory for this class to store a knowledge base.
```

Comment:
Update @param comment: it's really the directory where the knowledge base is stored which is subtly different than a directory to store a knowledge base. Basically, you're not writing (i.e. storing) a knowledge base.